src/setup/advanced_security.yml

```diff
@@ -16,15 +16,18 @@ pr:
 
 
 variables:
-  # Valid values for Language are: csharp,cpp,go,java,javascript,python,ruby
-  # Multiple languages can be separated by a comma.
-  advancedsecurity.codeql.language: --REPLACE--
   # advancedsecurity.codeql.logLevel: 2
   # advancedsecurity.codeql.debug: true
 
 steps:
   - task: AdvancedSecurity-Codeql-Init@1
     displayName: Initialize CodeQL
+    inputs: 
+      # Valid values for Language are: csharp,cpp,go,java,javascript,python,ruby
+      # Multiple languages can be separated by a comma.
+      languages: --REPLACE--
+      ram: 5120
+      threads: 0
```

Comment:
```suggestion
```

These are now defaulted properly by the task calculated by the os resources and no longer need to be hardcoded.